src/services/completions.ts

```diff
@@ -317,10 +317,20 @@ namespace ts.Completions {
         });
     }
 
-    const completionNameForLiteral = (literal: string | number | PseudoBigInt) =>
-        typeof literal === "object" ? pseudoBigIntToString(literal) + "n" : JSON.stringify(literal);
-    function createCompletionEntryForLiteral(literal: string | number | PseudoBigInt): CompletionEntry {
-        return { name: completionNameForLiteral(literal), kind: ScriptElementKind.string, kindModifiers: ScriptElementKindModifier.none, sortText: SortText.LocationPriority };
+    function completionNameForLiteral(literal: string | number | PseudoBigInt, preferences?: UserPreferences): string {

```

Comment:
You want `preferences` to be passed from all the calling locations..

Reply:
@sheetalkamat Thanks for the feedback 👍